js/client-ladder.js

```diff
@@ -135,22 +139,30 @@
 				var self = this;
 				this.$el.html('<div class="ladder pad"><p><button name="selectFormat"><i class="fa fa-chevron-left"></i> Format List</button></p><p><em>Loading...</em></p></div>');
 				if (app.localLadder) {
-					app.send('/cmd laddertop ' + format);
+					app.send('/cmd laddertop ' + format + (this.curSearchVal ? ' ,' + this.curSearchVal : ''));
 				} else {
 					$.get('/ladder.php', {
```

Comment:
On the other hand, I'm not sure we want to send a new HTTP request for every keypress. Unlike local searches, it probably _does_ make sense to either wait until the user presses Enter, or at least on a debounce.

Reply:
(Especially not an HTTP request that kicks off a slow SQL query.)

Reply:
I would explicitly wait until the user presses Enter, actually. With a "Filter" submit button.